src/app/api/v1/medications/schedule/route.ts

```diff
@@ -0,0 +1,27 @@
+import MedicationService from "@/services/medication";
+import { MedicationSchedule } from "@/types/medication";
+import { verifyUser } from "@/utils/auth";
+import { handleError } from "@/utils/errorHandler";
+import ERRORS from "@/utils/errorMessages";
+import { validateRoutes } from "@/utils/validateRoute";
+import { NextRequest, NextResponse } from "next/server";
+
+const route = "/api/v1/medications/schedule";
```

Comment:
```suggestion
const route = "/api/v1/medications/[userId]/schedule";
```

Could we update the route like so? Be sure to update the file name and the searchParams logic to account for this. The only query param should be date. Sorry, i shouldve specified this in the ticket.